qa-tests-backend/src/test/groovy/CSVTest.groovy

```diff
@@ -281,46 +305,18 @@ class CSVTest extends BaseSpecification {
             assert csvCVEs.get(i) == graphQLCVEs.get(i)
         }
 
-        where :
+        where:
         "Data is"
 
-        testLabel \
-          | baseGraphQLQuery                 | postgresGraphQLQuery \
-          | graphQLPayload | csvQuery
-        "Fixable Cves In Image Query" \
-          | FIXABLE_CVES_IN_IMAGE_QUERY      | FIXABLE_CVES_IN_IMAGE_POSTGRES_QUERY \
-          | [
-                id        : "sha256:e18c5814a9f7ddd5fe410f17417a48d2de562325e9d71337274134f4a6654e3f",
-                query: "",
-                // must scope without scope query since graphQL is hitting sub-resolver
-                scopeQuery: "",
-                vulnQuery : "Fixable:true",
-                vulnPagination: new Pagination(0, 0, new SortOption("cvss", true)),
-        ] | getCVETypeImageQuery() +
-            "Image Sha:sha256:e18c5814a9f7ddd5fe410f17417a48d2de562325e9d71337274134f4a6654e3f+Fixable:true"
-        "Fixable Cves In Component Query" \
-          | FIXABLE_CVES_IN_COMPONENT_QUERY  | FIXABLE_CVES_IN_COMPONENT_POSTGRES_QUERY \
-          | [
-                // openssl 1.0.1k-3+deb8u5
-                id        : getComponentId(),
-                query: "",
-                scopeQuery: "",
-                vulnQuery : "Fixable:true",
-                vulnPagination: new Pagination(0, 0, new SortOption("cvss", true)),
-        ] | getCVETypeImageQuery() + getComponentQuery()
-        "Fixable Cves In Deployment Query" \
-          | FIXABLE_CVES_IN_DEPLOYMENT_QUERY | FIXABLE_CVES_IN_DEPLOYMENT_POSTGRES_QUERY \
-          | [
-                id        : CVE_DEPLOYMENT.deploymentUid,
-                query: "",
-                scopeQuery: "",
-                vulnQuery : "Fixable:true",
-                vulnPagination: new Pagination(0, 0, new SortOption("cvss", true)),
-        ] | getCVETypeImageQuery() + "Deployment ID:${CVE_DEPLOYMENT.deploymentUid}+Fixable:true"
+        name                               | id                           | query
+        "FIXABLE_CVES_IN_IMAGE_QUERY"      | IMAGE_SHA                    | "Image Sha:${IMAGE_SHA}+Fixable:true"
+        "FIXABLE_CVES_IN_COMPONENT_QUERY"  | componentId                  | componentQuery
```

Comment:
shouldn't the parameters here be `getComponentId()` and `getComponentQuery()` ? Or does some groovy magic make the link automatically ?

Reply:
groovy magic that converts getters into properties https://groovy-lang.org/objectorientation.html#_fields_and_properties

Reply:
Let's look at this example https://ideone.com/X3IZ8S#stderr

```groovy
class Ideone {
	static class A {
		String s = "A"
		String getS() {
			return "Getter" + s
		}
		String getX() {
			return "Getter X"
		}
	}
	static class B {
		String s = "B"
	}
    static void main(String[] args) {
		A a = new A()
		println a.s
		println a.getS()
		println a.x
		B b = new B()
		println b.s
		
    }
}
```

```
GetterA
GetterA
Getter X
B
```